src/jqLite.js

```diff
@@ -587,6 +587,9 @@ forEach({
   },
 
   attr: function(element, name, value) {
+    if (element.nodeType === 2 || element.nodeType === 3 || element.nodeType === 8) {
```

Comment:
* please use `NODE_TYPE_TEXT` and `NODE_TYPE_COMMENT`, for nodeType === 2, please create `NODE_TYPE_ATTRIBUTE` to src/Angular.js and use it
* add `var nodeType = element.nodeType` and do the comparison to it